src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/InitialChecksService.kt

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service
+
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.EligibilityCheckDetails
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.EligibilityState
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.InitialChecks
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.SuitabilityCheckDetails
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.SuitabilityState
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.EligibilityCheck
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.SuitabilityCheck
+
+@Service
+class InitialChecksService(val offenderService: OffenderService, val policyService: PolicyService) {
+
+  @Transactional
+  fun getCurrentInitialChecks(prisonNumber: String): InitialChecks {
+    val currentAssessment = offenderService.getCurrentAssessment(prisonNumber)
```

Comment:
wasn't super sure about reusing the offenderService.. I could have reimplemented a lot of the same logic but it seemed very convenient. Seems like the kind of thing that could be annoying in the future